installer/pkg/components/rabbitmq/constants.go

```diff
@@ -10,6 +10,7 @@ import (
 
 const (
 	Component         = "rabbitmq"
+	ComponentAlias    = "messagebus"
```

Comment:
This was a workaround, which we also do in the helm charts too, so that `messagebus` can be used as part of the rabbitmq role binding for the label and service account, and thus allow the messagebus workload to be admitted by the pod security policy.